frontend/components/Header.tsx

```diff
@@ -55,7 +55,7 @@ export function Header({
           {isHomepage ? <div></div> : (
             <a
               href="/"
-              class="outline-none focus-visible:ring-2 ring-cyan-700"
+              class="outline-none focus-visible:ring-2 ring-jsr-cyan700"
```

Comment:
I think this one might be a typo (shouldn't there be a hyphen between, as in `cyan-700`?)